include/miroil/miroil/edid.h

```diff
@@ -34,31 +34,13 @@ struct Edid
     PhysicalSizeMM size{0,0};
 
     struct Descriptor {
-        enum class Type : uint8_t {
-            timing_identifiers = 0xfa,
-            white_point_data = 0xfb,
-            monitor_name = 0xfc,
-            monitor_limits = 0xfd,
-            unspecified_text = 0xfe,
-            serial_number = 0xff,
-
-            undefined = 0x00,
-        };
-
-        union Value {
-            char monitor_name[13];
-            char unspecified_text[13];
-            char serial_number[13];
-        };
-
-        Type type{Type::undefined};
-        Value value{{0}};
-
-        std::string string_value() const;
+        std::string string_value() const { return {}; };
     };
-    Descriptor descriptors[4];
 
-    Edid& parse_data(std::vector<uint8_t> const&);
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wpedantic"
+    Descriptor descriptors[0];
+#pragma GCC diagnostic pop
```

Comment:
This breaks the miroil ABI - we should bump the .soname

Reply:
We're already bumping in 2.22: 974b779b8af92ff985693d7bd045eaffb9cbe124

But I did clear the removed symbol from the map and `.symbols` file.